### ISSUE TITLE
Community Patch Bug

### ISSUE THREAD

**Issue by ping827** [Tyler-IN/MnB2-Bannerlord-CommunityPatch#242]
https://imgur.com/gallery/7g31mF0

Let’s me load into my saved game if I turn community patch off on launcher

**Comment by fenrir0054** [comment]
I got the same thing when trying to load my save just now. I've been using 0.1.0-rc2 since the start of my campaign, was able to play and save and load just fine, however when closing the game coming back after a while and trying to load I got the same crash.

Interestingly, a workaround is to start a new campaign and immediately click back (before selecting a cultural background) and then load the save. This will let me load the save and play.

**Comment by famaouzi** [comment]
> Interestingly, a workaround is to start a new campaign and immediately click back (before selecting a cultural background) and then load the save. This will let me load the save and play.

This works XD

**Comment by ping827** [comment]
Genius it works

**Comment by Tyler-IN** [comment]
Yeah some of our patches account for this, some don't - had to do w/ the way the campaign initializes static data when starting a new one vs. loading a game. We'll fix this.

**Comment by GuilhermeDaniluski** [comment]
> I got the same thing when trying to load my save just now. I've been using 0.1.0-rc2 since the start of my campaign, was able to play and save and load just fine, however when closing the game coming back after a while and trying to load I got the same crash.
> 
> Interestingly, a workaround is to start a new campaign and immediately click back (before selecting a cultural background) and then load the save. This will let me load the save and play.

Oh god this really works
